resources/keywords/dbaas_policy.resource

```diff
@@ -100,16 +103,91 @@ Click If Element Present
     END
 
 User Creates DBaaSPolicy For ${namespace} From Admin View As ${DBaaSPersona}
+    Set Test Variable   ${operatorNamespace}    ${namespace}
     User Navigates To DBaaSPolicy Screen    ${namespace}
-    Click If Element Present    ${lblOperand}
-    Wait Until Page Contains Element    ${btnCreateDBaaSPolicy}
-    Click Element    ${btnCreateDBaaSPolicy}
-    Create Custom DBaaSPolicy as ${DBaaSPersona}
+    User Creates DBaaSPolicy For ${DBaaSPersona}
 
 User Creates DBaaSPolicy For ${namespace} From Invalid Namespace Alert on Dev View As ${DBaaSPersona}
+    Set Test Variable   ${operatorNamespace}    ${namespace}
     User Navigates To Import MongoDB Provider Account On Developer View
     Wait Until Page Contains Element    ${errCreateDBaaSPolicy}          timeout=20s
     Click Element    ${errCreateDBaaSPolicy}
     Create Custom DBaaSPolicy as ${DBaaSPersona}
     User Selects Administrator View
 
+User Creates DBaaSPolicy For ${DBaaSPersona}
+    Click If Element Present    ${lblOperand}
+    Wait Until Page Contains Element    ${btnCreateDBaaSPolicy}
+    Click Element    ${btnCreateDBaaSPolicy}
+    Create Custom DBaaSPolicy as ${DBaaSPersona}
+
+Max Quota Exceeded Error Displays
+    Scroll Element Into View    ${txtQuotaError}
+    Element Should Be Visible   ${txtQuotaError}
+
+Specify Connection Namespaces And Labels
+    ${ns}=  Run Keyword And Return Status   Should Contain    ${nsAndLabel}    namespace
+    ${label}=  Run Keyword And Return Status   Should Contain    ${nsAndLabel}    label
+    IF      ${ns}
+        Click Element    ${btnToggleConnNS}
+        Add Policy Connection Namespaces    ${addNS}
+    END
+    IF    ${label}
+        Click Element    ${btnToggleConnNSSelect}
+        Element Should Be Visible    ${btnToggleMatchExp}
+        Click Element    ${btnToggleMatchExp}
+        Element Should Be Visible    ${btnAddMatchExp}
+        Add Policy Connection Nsselector    ${addLabel}
+    END
+
+User Creates DBaaSPolicy With ConnectionNamespace For ${namespace} As ${DBaaSPersona}
+    Set ConnectionNamespace For DBaaSPolicy
+    User Creates DBaaSPolicy For ${namespace} From Admin View As ${DBaaSPersona}
+
+Set ConnectionNamespace For DBaaSPolicy
+    Set Test Variable   ${nsAndLabel}   namespace
+    Set Test Variable   @{addNS}    ${connectionNS}
+
+DBaaSInventory Is Available For ${targetns}
+    User Filters Project ${targetns} On Project DropDown And Navigates To Database Access Page
+    Imported Provider Account Available On Database Access Screen
+    User Navigates To Add ${DBProvider} To Topology Screen
+    Select Given Provider Account On Add Topology Screen
```

Comment:
Why not use ${DBProvider} in this line too?

Reply:
Existing keyword "Select Given Provider Account On Add Topology Screen" uses variable assignment internally. Please go through the definition and let me know if still this needs to be addressed.

Reply:
OK got it.